lib/src/components/NxCombobox/NxCombobox.scss

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use '../../scss-shared/nx-container-helpers';
+
+ .nx-combobox {
+   display: inline-block;
+   position: relative;
+ 
+   &__menu {
+     position: fixed;
+     right: 100%;
+     width: 100%;
+ 
+     &--error {
+       padding: 0;
+     }
+   }
+ 
+   &__empty-message {
+     @include nx-container-helpers.container-horizontal;
+ 
+     padding: var(--nx-spacing-base) var(--nx-spacing-4x);
+     text-align: center;
+   }
+ 
+   &--dropdown-showable{
+     .nx-combobox__menu {
+       position: absolute;
+       right: initial;
+
+       .nx-combobox__option--visual-selected {
```

Comment:
We actually usually call this just `selected`

Reply:
Updated here c6da513589d58bebdd5113d72da837ae895ba74e.